### DISCUSSION TITLE
Branch creation template for specific subject

### DISCUSSION THREAD

**Discussion by Viddesh1** [tusuii/All-Pracs#21] [Ideas]
Hello @tusuii,

Since, We know that for adding the code for the new subject. A dedicated branch name specific to that subject is to be created in the main remote repository of your codebase.

A specific template should be helpful for creating a branch in your remote repository. So, that the contributor can just push their pull request for that specific branch only.

What do you think? 🤔

It will save the manual work too for creating a specific branch for a specific subject

**Comment by tusuii** [comment]
@Viddesh1 
Great idea, how should we can do that any idea?
Regards
@tusuii

**Reply by Viddesh1** [reply]
@tusuii 

I am not yet sure of how it can be implemented.

 I think something like an issue template. Where a contributor can click on the issue button and it pops up different choices are given to choose from like "add a subject codes" or "report issue". 

We may need to dig deep into how it can actually be done.

**Reply by tusuii** [reply]
@Viddesh1 
Kool really nice idea

**Comment by Viddesh1** [comment]
🏁  This discussion topic is based on the experience that we have in issue #17. Adding it as a reference.
